apps/web/src/components/layout/components/HeaderNav.tsx

```diff
@@ -13,6 +13,7 @@ import { Tooltip } from '../../../design-system';
 import { INTERCOM_APP_ID } from '../../../config';
 import { SpotlightContext } from '../../../store/spotlightContext';
 import { HEADER_HEIGHT } from '../constants';
+import { LimitBar } from '../../../pages/integrations/components/LImitBar';
```

Comment:
```suggestion
import { LimitBar } from '../../../pages/integrations/components/LimitBar';
```
Filename spelling. 🙂